pages/people.md

```diff
@@ -42,7 +42,8 @@ based on commits.
 | [Ravikrishna Adepu](https://github.com/adepuravikrishna) | [SAP Labs LLC](https://sap.com)     | [PT][pt]   |
 | [Russell Carter Jr](https://github.com/rcarterjr)        | [ASRC Federal](https://www.asrc.com)| [ET][et]   |
 | [Sri Kailash](https://github.com/srikailash)		         | [Booking.com](https://booking.com)  | [CEST][cst]|
-| [Tanisha Faulkner] (https://github.com/plainolneesh)     | [ASRC Federal](https://www.asrc.com)| [ET][et]   |
+| [Tanisha Faulkner](https://github.com/plainolneesh)     | [ASRC Federal](https://www.asrc.com)| [ET][et]   |
+| [Arvind Shyamsundar](https://github.com/arvindshmicrosoft/) | [Microsoft](https://www.microsoft.com) | [PT][pt] |
```

Comment:
I *think* this list is supposed to be alphabetical, if you want to put your name at the top, in order, instead of here. :smiley_cat:

Reply:
Done, thanks for pointing that out. Also adjusted the Microsoft hyperlink to suit.